apishared/const.go

```diff
@@ -54,5 +54,5 @@ const LongReadTimeout = 500 * time.Millisecond
 var KoopmanTable = crc32.MakeTable(crc32.Koopman)
 var ReadBufferSize = 8192
 
-// MaxBufSize is used to
-const MaxBufSize = 2048
+const FileServiceMaxBufSize = 2048
```

Comment:
I think that's probably too small for a real disk...  I'd probably up that to 8192, although I don't really have a good reason for it.